app/src/test/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegrationTest.kt

```diff
@@ -107,6 +108,24 @@ class BrowserToolbarIntegrationTest {
         verify(browserToolbarIntegration).observerSecurityIndicatorChanges()
     }
 
+    @Test
+    fun `WHEN start method is called THEN observe erase tabs CFR changes`() {
+
+        doNothing().`when`(browserToolbarIntegration).observeEraseCfr()
+
+        browserToolbarIntegration.start()
+
+        verify(browserToolbarIntegration).observeEraseCfr()
+    }
+
+    @Test
+    fun `WHEN stop method is called THEN erase tabs scope is not active anymore`() {
+
```

Comment:
nit: Extraneous empty line.
Separating the 3 main sections: arrange, act, assert would be nice though.